features/facebookImport/src/importer/importer-util.js

```diff
@@ -4,11 +4,20 @@ import {
     MissingFileImportException,
 } from "./failed-import-exception";
 
-async function readJSONFile(dataFileName, zipFile) {
+async function relevantZipEntries(zipFile) {
     const entries = await zipFile.getEntries();
-    const dataZipEntry = entries.find((entryName) =>
-        entryName.includes(dataFileName)
+    return entries.filter(
```

Comment:
The reasoning there is the same as above. Indeed a different way to structure this could be to maybe add an accessor like `relevantEntries` to the `ZipFile` object